en/editor/assets/index.md

```diff
@@ -111,6 +111,28 @@ The search function is a combined function that can limit the search type and sp
 - Shortcut key `Esc` to cancel rename
 - In addition, the initial name of the Typescript asset will be treated as its `className`, and the `className` cannot be repeated.
 
+### Exporting Assets Package
+
+Select the resource to be exported and right click on it, then select **Export Resource Package**. The **Export Resource** panel will automatically list the currently selected resource and its dependencies. If you don't need to export dependent resources, you can uncheck **Include Dependent Resources** in the bottom left corner of the **Export Resources** panel.
+
+Once you have identified the resources to be exported, click the **Export** button to bring up the file storage dialog box where you will need to specify a folder location and file name.
+
+![exporting](img/package-asset/exporting.png)
+
+> **Note**: Exported zip packages are only suitable for **importing resource packages** use, as described below.
+
+### Importing Assets Package
+
+Select and right click on the folder you want to import resources from, or right click on the **blank space** in the **Export Explorer** panel, then select **Import Package** and select the zip package exported from **Export Package** in the file browse dialog box. The resources in the zip package will be automatically parsed into the **Import Package** panel that pops up.
+
+The import process will also ask the user to reconfirm the imported resources, so you can uncheck some resources to remove the ones you don't want to import.
+
+![importing](img/package-asset/importing.png)
+
+> **Note**:
+> 1. Imported zip packages, only zip packages exported by Cocos Creator 3.0.0 and above using **Export Resource Packages** are supported.
```

Comment:
Cocos Creator 3.0.0
-->
Creator

We dont need to specify v3 since we are in the v3 docs when reading this.